src/linters/swift-format-lockwood.js

```diff
@@ -39,8 +39,7 @@ class SwiftFormatLockwood {
 		}
 
 		const fixArg = fix ? "" : "--lint";
-		return run(`${prefix} swiftformat ${fixArg} ${args} "."`, {
-			dir,
+		return run(`${prefix} swiftformat ${fixArg} ${args} ${dir}`, {
```

Comment:
Documentation for `[linter]_dir`: "Directory where the linting command should be run". This would change that behavior. I think the command should be left the way it is.

Reply:
Fixed back to the original format